promises.cf.in

```diff
@@ -138,7 +138,7 @@ bundle common inventory
 # Tested to work properly against 3.5.x
 {
   classes:
-      "other_unix_os" expression => "!windows.!macos.!linux.!freebsd";
+      "other_unix_os" expression => "!windows.!macos.!linux.!freebsd.!aix";
```

Comment:
I commented in the ticket but here as well for convenience ...

Instead of adding on `.!aix` (and not aix), what you you think about changing it to not windows or linux or freebsd ( `!(windows|linux|freebsd|aix)` )?

    classes:
     "other_unix_os" expression => "!(windows|macos|linux|freebsd|aix)";

Reply:
no problem...looks indeed better.

Reply:
Thanks